Header header
point goalPoint
float64 totalCost
float64 totalTime
path[] pathAry
